man/tlp.8

```diff
@@ -102,7 +102,17 @@ tlp recalibrate BAT0
 .SH FILES
 .I /etc/default/tlp
 .RS
-System-wide configuration file containing all power saving settings.
+System-wide default configuration file containing all power saving settings.
+.RE
+.PP
+.I /etc/tlp.conf.d/
+.RS
+Custimized system-wide configuration files which will override /etc/default/tlp.
+.RE
+.PP
+.I /run/tlp/tlp.conf
+.RS
+Runtime system-wide configuration file which is the final setting.
```

Comment:
This file is a result and never read by TLP itself, so what about:

> Runtime system-wide file which reflects the final settings aggregated from all configuration files.

Reply:
this is to response to the concern that 3-party utility might need to read a single file for all configuration.
https://github.com/linrunner/TLP/issues/398#issuecomment-475742618

Reply:
@linrunner 
it's not necessary but just to let other utility and people have a convenient way to check all setting by a single file. How do you think about it? Or we can still remove it if you think removing it will be better.